pkg/graph/schema/types/possible_operational_solution.graphql

```diff
@@ -4,6 +4,7 @@ type PossibleOperationalSolution {
   key: OperationalSolutionKey!
   treatAsOther: Boolean!
   pointsOfContact: [PossibleOperationalSolutionContact!]!
+  primaryContact: PossibleOperationalSolutionContact

```

Comment:
I'd say we should either:
- Ensure this resolver uses a dataloader
- Remove this resolver and leave it up to the frontend to query `pointsOfContact` where `isPrimary: true` to fetch primaries

This will currently fire off a SQL query to fetch the primary contact for each (less than ideal, but not a dealbreaker I'd say)

Reply:
The general desired paradigm we've been migrating towards is to include resolvers in this fashion. I considered this as being a dataloader, but it seems the efficiency gains for a dataloader are more about batch loading data. Wouldn't we need to fire a SQL query regardless of which solution we use?

Another consideration, if the frontend doesn't need the primaryContact field can't they simply omit it from the GQL query?

Reply:
I'm fine to leave it, and the frontend can use whichever option makes the most sense for them! In any case, the query made is generally simple & quick!